v5_asset_service.go

```diff
@@ -476,3 +477,40 @@ func (s *V5AssetService) GetAllCoinsBalance(param V5GetAllCoinsBalanceParam) (*V
 
 	return &res, nil
 }
+
+type V5WithdrawResponse struct {
+	CommonV5Response `json:",inline"`
+	Result           V5WithdrawResult `json:"result"`
+}
+
+type V5WithdrawResult struct {
+	ID string `json:"id"`
+}
+
+type V5WithdrawParam struct {
+	Coin        Coin        `json:"coin"`
+	Chain       string      `json:"chain"`
+	Address     string      `json:"address"`
+	Tag         *string     `json:"tag,omitempty"`
+	Amount      string      `json:"amount"`
+	Timestamp   int64       `json:"timestamp"`
+	ForceChain  bool        `json:"forceChain"`
+	AccountType AccountType `json:"accountType"`
+	FeeType     int         `json:"feeType"`
+	RequestID   *string     `json:"requestId,omitempty"`
+}
```

Comment:
nits:

If param is required, it should be a pointer, otherwise it should be a value.
To specify whether it is required or not.

ref.
https://github.com/hirokisan/bybit/blob/2c4c0e0b93770a9417bea6d4def62e8b2ccde716/v5_order_service.go#L25-L57